.github/workflows/ci-model-regression.yml

```diff
@@ -692,7 +692,7 @@ jobs:
           echo "::set-output name=result::$OUTPUT"
 
       - name: Publish results as a PR comment
-        uses: marocchino/sticky-pull-request-comment@v2.1.1
+        uses: marocchino/sticky-pull-request-comment@39c5b5dc7717447d0cba270cd115037d32d28443 # v2.2.0
```

Comment:
Intentional upgrade? (also other uses of this action have been upgraded)

Reply:
Yeah I've intentionally upgraded this one - dependabot updated it recently and it pinned it to a version tag that didn't have a signed commit, where other tags did. Just seemed a bit weird so I bumped it to the next signed released tag.